_data/sidenav.yml

```
@@ -3,5 +3,11 @@ main:
   - { url: '/getting-started', text: 'Getting Started' }
   - { url: '/step-by-step', text: 'Step-by-Step Guide' }
   - { url: '/nifi-features', text: 'NiFi Features' }
-  - { url: '/cookbook-recipes', text: 'Cookbook Recipes' }
+  - { url: '/cookbook-recipes', text: 'Cookbook Recipes',
+      childPages: [
+        { url: '/get-data-from-a-relational-database', text: 'Relational Database' },
```

I think a consistent naming pattern would be useful

```suggestion
        { url: '/get-data-from-a-relational-database', text: 'Relational Database to MarkLogic' },
```